packages/@o3r/dev-tools/src/utils/check-peer-deps.ts

```diff
@@ -67,6 +67,7 @@ export function getPackagesToInstallOrUpdate(packageName: string) {
 /**
  * Log an instruction with the packages to install or update to match a package peer dependencies
  *
+ * @deprecated use `checkPackagesRule` or `checkPackagesToInstallOrUpdate` functions exposed by `@o3r/schematics` instead. Will be removed in Otter v10
```

Comment:
I think this method is only used in `packages/@o3r/schematics/src/rule-factories/check-packages-peers/index.ts` where you replaced it with the new one. In this case, why not removing it completely from here?

Reply:
Because it is exposed by the package, so removing it would be a breaking change (even if on our side we you it only once :S)